packages/api/src/env.js

```diff
@@ -94,6 +98,20 @@ export function envAll (req, env, ctx) {
     release: env.SENTRY_RELEASE,
     pkg
   })
+
+  // Attach a `Logging` instance, which provides methods for logging and writes
+  // the logs to LogTail. This must be a new instance per request.
+  // Note that we pass `ctx` as the `event` param here, because that's what it
+  // actually is, we seem to have just misnamed it "ctx".
+  env.log = new Logging(req, ctx, {
```

Comment:
Can we add log to the env type definition?

Reply:
@adamalton a todo!

Reply:
Good spot, thanks. Done.